docs/learn-legitimate-projects.md

```diff
@@ -1,242 +1,162 @@
-You can learn more about crowdloans [here](https://kusama.network/auctions) and on our
-[wiki](https://guide.kusama.network/docs/mirror-learn-crowdloans).
-
-A legitimate team will never ask you to send them funds in order to participate in a crowdloan. That
-being said, some teams have been doing token sales or swaps in an attempt to get a head start in
-raising funds for the upcoming auctions, but these are not crowdloans and they imply full trust in
-the team. So, if you plan to participate in these token swaps make sure the project is reputable and
-that you're getting the correct information through their official site and social media channels.
-
-Similarly, several centralised exchanges are creating ways to participate in crowdloans through
-their platforms, but also some wallets are integrating crowdloan functionality to their apps and
-more are sure to follow. Any legitimate effort should be using the native crowdloans module in the
-background and simply offer a more streamlined experience for the user. Making sure that's the case
-is necessary before using these services, but in any case it still involves trusting the exchange or
-the service provider.
+With the recent launch of parachains on Kusama, many projects that aim to become a parachain launched a crowdloan to gather the necessary funds to participate in the parachain auctions. But with all the buzz around the Kusama parachain launch and the imminent Polkadot launch, many scams may also surface. So, crowdloans need their own section to make sure that you're participating safely.
+
+First of all, only projects that aim to become a parachain should have a crowdloan. If a project isn't a parachain candidate, there shouldn't be a crowdloan associated with it.
+
+The surest way to participate in a parachain crowdloan is the native way through the [Crowdloan module](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fkusama-rpc.polkadot.io#/parachains/crowdloan) on polkadot.js.org. This issues a special extrinsic that locks your funds until the parachain slot lease period ends and guarantees that you'll get your stake back after that. You can learn more about crowdloans [here](https://kusama.network/auctions) and on our [wiki](https://guide.kusama.network/docs/mirror-learn-crowdloans).
+
+Many parachain candidates offer a way to participate through their site as well. But you should ensure that they are using the crowdloan pallet in the background, and they're simply wrapping that in a nicer, more user-friendly UI. If instead, their crowdloan interface *transfers* funds to an account, these funds will be totally under their control, and this means you need to fully trust that the team will use the funds for the crowdloan, return your share to you when the lease period ends or if they don’t get a spot, and will secure them properly! And if they're doing something like that, it should be explicitly mentioned in their site and documentation.
```

Comment:
`...return your share to you when the lease period ends or if they don’t get a spot, and will secure them properly!`
- We should avoid "!" unless absolutely necessary.

`And if they're doing something like that, it should be explicitly mentioned in their site and documentation.`
- take out "And".